clang/test/SemaSYCL/sycl-device-reqd-sub-group-size-template.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsyntax-only -verify -pedantic %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsyntax-only -ast-dump -verify -pedantic %s | FileCheck %s
```

Comment:
Isn't second command covers both diagnostics and AST checks?

Reply:
Thanks @bader.  Yes, the second command covers both diagnostics and AST checks. 
First RUN command is not needed and this is removed now.